rosetta-source/src/main/rosetta/observable-asset-type.rosetta

```diff
@@ -184,9 +184,19 @@ type Price extends PriceSchedule: <"Specifies a price as a single value to be as
     condition AmountOnlyExists: <"The amount must exist when the price represents a single value, and steps must be absent.">
         value exists and datedValue is absent
 
+// ART02 Observable and Index refactoring
```

Comment:
same comment as prevous one above 

**PARTIALLY AGREE**

- AGREE with new type Observable and current list of attributes
- REMARK beleive there is missing one attribute in the list

**PROPOSAL**

- add one addional attribute for Observable that would be of type **FixedRate**
- RATIONALE : see details in previous comment